src/app/pages/vm/devices/device-edit/device-edit.component.ts

```diff
@@ -254,14 +258,17 @@ export class DeviceEditComponent implements OnInit {
         {
           name : 'DISK_sectorsize',
           placeholder : 'Disk sector size',
-          tooltip : 'Enter the sector size in bytes. The default <i>0</i>\
-                     leaves the sector size unset.',
+          tooltip : 'A fixed amount of data that is the minimum storage\
+                     unit of a hard drive. Enter the sector size in\
```

Comment:
Reverse the two sentences about the default and entering a value, or it sounds like the user is commanded to enter a value.  "the default is the best to use. if a custom size is needed, enter the value in bytes..."